tools/pot/tests/test_data_generation.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+from addict import Dict
+
+import pytest
+
+from openvino.tools.pot.data_loaders.creator import create_data_loader
+from openvino.tools.pot.graph import load_model
+from openvino.tools.pot.graph.model_utils import get_nodes_by_type
+
+
+TEST_MODELS = [
+    ('mobilenet-v2-pytorch', 'pytorch', None, None),
+    ('mobilenet-v2-pytorch', 'pytorch', None, (3, 640, 720)),
+    ('mobilenet-v2-pytorch', 'pytorch', 'HWC', (224, 224, 3)),
```

Comment:
Shouldn't be additionally tested for TF models with the NHWC layout?

Reply:
SyntheticImageLoader uses only `H` and `W` dimensions. Batch dimension will be ignored. I can add test to verify this case